webassembly/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM gitpod/workspace-full
+
+RUN bash -cl "cargo install wasm-pack cargo-wasm cargo-generate \
+    && curl \"https://wasmtime.dev/install.sh\" -sSf | bash; \
+       rustup target add wasm32-wasi"
+
+RUN wget "https://github.com/CraneStation/wasi-sdk/releases/download/wasi-sdk-8/wasi-sdk_8.0_amd64.deb" \
+    && sudo dpkg -i wasi-sdk_8.0_amd64.deb
+
+RUN git clone "https://github.com/emscripten-core/emsdk.git" $HOME/.emsdk \
+    && cd $HOME/.emsdk \
+    && ./emsdk install latest \
+    && ./emsdk activate latest \
+    && bash -c "source ./emsdk_env.sh"
+
+RUN printf "\nsource $HOME/.emsdk/emsdk_env.sh\nclear\n" >> ~/.bashrc
+
+RUN brew install binaryen
+
+ENV PATH="/opt/wasi-sdk/bin:$PATH"
```

Comment:
Oh, where did the `/opt/wasi-sdk` suddenly come from? Please move this command right after the command that creates `/opt/wasi-sdk`, for clarity.